templates/components/faceted-search/facets/hierarchy.html

```diff
@@ -3,7 +3,7 @@
     <div
         class="accordion-navigation toggleLink {{#unless ../start_collapsed }} is-open {{/unless}}"
         role="button"
-        data-collapsible="#facetedSearch-content--{{hyphenate facet }}">
+        data-collapsible="#facetedSearch-content--{{#replace '&' (hyphenate facet) }}{{/replace}}">
```

Comment:
I really don't like the idea of doing special characters replaces at the handlebars level. Today we are doing `&` tomorrow it is going to be some other special characters.

We should either strip it or escape it at the backend level. Right now

Reply:
You can't escape `&` to be placed in a fragment. There are no other special characters like this.

Reply:
Since we are just doing a replace for & in the string for the fragment can't we do that on the backend itself ?